cocotb/share/lib/embed/gpi_embed.cpp

```diff
@@ -346,7 +346,8 @@ int embed_sim_init(gpi_sim_info_t *info)
     }
 
     // Set language in use as an attribute to cocotb module, or None if not provided
-    const char *lang = getenv("TOPLEVEL_LANG");
+    const char *lang;
+    lang = getenv("TOPLEVEL_LANG");
     PyObject *PyLang;
```

Comment:
If you wrapped everything from line 349 to line 369 in a set of braces, then this change wouldn't be needed, and it would nicely show that these variables are gone before reaching the cleanup.

Reply:
I had originally done this, but changed it to prefer a smaller diff, but I tend to think getting local as possible with variable declarations is a good idea.

Reply:
The diff is still small if you tell github to ignore whitespace